chart/values.schema.json

```
@@ -1737,6 +1737,24 @@
                         }
                     }
                 },
+                "resources": {
+                    "description": "Resources for the migrate database job pod",
+                    "type": "object",
+                    "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.19.0/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements",
```

```suggestion
                    "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.22.0/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements",
```
We support a newer version.

---

See: https://github.com/apache/airflow/blob/d70909422ec7ec2f6e5311d26c1b15e31c3bc188/chart/tests/helm_template_generator.py#L34-L35